File: lib_ebook_api/src/main/java/com/ebook/api/entity/LoginDTO.java
package com.ebook.api.entity;

import com.ebook.api.entity.User;

import androidx.annotation.NonNull;

public class LoginDTO {
    private User user;
    private String token;

    @NonNull
    @Override
    public String toString() {
        return "LoginDTO{" +
                "user=" + user +
                ", token='" + token + '\'' +
                '}';
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
}

File: lib_ebook_api/src/main/java/com/ebook/api/entity/LoginDTO.kt
package com.ebook.api.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * 登录/刷新响应载荷（服务端 TokenPair：双 token + 用户信息）。
 */
@Serializable
data class LoginDTO(
    var user: User? = null,
    var token: String? = null,
    // 线上键为 refresh_token：Kotlin 属性保持驼峰，边界翻译由 @SerialName 完成
    @SerialName("refresh_token")
    var refreshToken: String? = null
)
